src/security/intel/txt/ramstage.c

```diff
@@ -202,6 +222,13 @@ static void push_sinit_heap(u8 **heap_ptr, void *data, size_t data_length)
 	}
 }
 
+static bool is_txt_server_capable(void)
+{
+	msr_t msr = rdmsr(MSR_BOOT_GUARD_SACM_INFO);
+
+	return !!(msr.hi & (B_BOOT_GUARD_SACM_INFO_TXT_CAPABILITY >> 32));
```

Comment:
```suggestion
	return !!(msr.raw & B_BOOT_GUARD_SACM_INFO_TXT_CAPABILITY);
```
I went to check constants and types because of the way this was written.

Reply:
Applied and squashed to https://github.com/Dasharo/coreboot/pull/685/commits/db665ed3b6469c3fbad3743c3c0752f4324f36ac